app/routes/events/view.js

```diff
@@ -9,7 +9,7 @@ export default Route.extend({
 
   model(params) {
     return this.store.findRecord('event', params.event_id, {
-      include: 'event-topic,event-sub-topic,event-type,event-copyright,tax,sponsors'
+      include: 'event-topic,event-sub-topic,event-type,event-copyright,tax,sponsors,speakers-call,microlocations,tracks,session-types'
```

Comment:
This route is the parent route of a whole bunch of other routes. So, it is critical we keep this as lean as possible.